web/client/components/style/EqualIntervalComponents/__tests__/ColorRampItem-test.jsx

```diff
@@ -0,0 +1,44 @@
+const React = require('react');
```

Comment:
tests are fine, 
please, add copyright section with current year (in every new file)